examples/resource-monitoring-app/resource-monitoring-common/src/instances/HepafilterMonitoring.cpp

```diff
@@ -0,0 +1,48 @@
+#include <app-common/zap-generated/attributes/Accessors.h>
+#include <app/util/config.h>
+#include <instances/HepaFilterMonitoring.h>
+
+using namespace chip::app::Clusters;
+using namespace chip::app::Clusters::HepaFilterMonitoring;
+using chip::Protocols::InteractionModel::Status;
+
+using namespace chip::app::Clusters;
+using namespace chip::app::Clusters::ResourceMonitoring;
+using chip::Protocols::InteractionModel::Status;
+
+//-- Hepa filter Monitoring instance methods
+CHIP_ERROR HepaFilterMonitoringInstance::AppInit()
+{
+    ChipLogDetail(Zcl, "HepaFilterMonitoringInstance::Init()");
+    return CHIP_NO_ERROR;
+}
+
+Status HepaFilterMonitoringInstance::OnResetCondition()
+{
+    ChipLogDetail(Zcl, "HepaFilterMonitoringInstance::OnResetCondition()");
+
+    if (GetDegradationDirection() == DegradationDirectionEnum::kDown)
```

Comment:
There may be situations were the actual condition can be read by the appliance and is not just a counter or something like this.
We were thinking about this as well, but then decided that it's application logic that should take care what should be reseted.

Reply:
I guess the real question is this: if degradation direction is down, is there any situation where after a reset the condition will not be 100?  Or change indication is not ok?

If not, or if those cases are rare and involve the app setting some specific values instead of leaving the old ones, then the cluster could reset itself to those values and then the app can override if it really wants to.  The only case that does not handle is if we want the old values to stay on reset....

Again, this is all a followup; worth filing an issue to track that.

Reply:
created this issue to track that https://github.com/project-chip/connectedhomeip/issues/27986

Reply:
I actually implemented it in this PR just now. If you want to have a look at that specifically it's this commit: https://github.com/project-chip/connectedhomeip/pull/27798/commits/d0af4d2e2afcf0b67a2a4e8e1a0741103ea2716c